.github/workflows/ci.yml

```diff
@@ -24,8 +25,12 @@ jobs:
         with:
           node-version: 18
 
-      - run: pip install -r src/python-fastui/requirements/all.txt
-      - run: pip install src/python-fastui
+      - name: Install the latest version of rye
+        uses: eifinger/setup-rye@v2
+
+      - run: make install
```

Comment:
oke almost done, but here somehow on github it's using python 3.8, disrespecting the matrix python version. 
I didn't commit .python_version, but that shouldn't be. Any idea how can I force rye to use the correct python version respecting matrix version?